designer/client/src/components/toolbarComponents/ToolbarsLayer.tsx

```diff
@@ -58,13 +58,13 @@ function ToolbarsLayer(props: { toolbars: Toolbar[]; configId: string }): JSX.El
     return (
         <DragDropContext onDragEnd={onDragEnd} onDragStart={onDragStart}>
             <SidePanel side={PanelSide.Left} className={cn(styles.left, isDragging && styles.isDraggingStarted)}>
-                <ToolbarsContainer availableToolbars={availableToolbars} side={ToolbarsSide.TopLeft} className={cn(styles.top)} />
-                <ToolbarsContainer availableToolbars={availableToolbars} side={ToolbarsSide.BottomLeft} className={cn(styles.bottom)} />
+                <ToolbarsContainer availableToolbars={availableToolbars} side={ToolbarsSide.TopLeft} />
```

Comment:
What was the reason for these all `className={cn(styles.bottom)}`? I didn't see any changes related to this class, but they are just removed

Reply:
fix